documentation/create_overview_doc/createdoc.sh

```diff
@@ -1,56 +1,27 @@
 #!/bin/bash
 #set -x
 
-# Log file for storing the token usage information
-log_file="bito_usage_log.txt" 
-
-# Directory containing prompt files for NLP tasks
-prompt_folder="AI_Prompts"  
-
-# Global variables for session token counts
+log_file="bito_usage_log.txt"
```

Comment:
**Suggestion**: While updating the script to include checks for Code2Flow and Graphviz's 'dot' command, consider adding error handling for cases where these tools are not installed. Providing a clear error message and guidance on installing these dependencies would improve the user experience. <br> **Code Suggestion**: 
 ```
 if ! command -v code2flow &> /dev/null; then
     echo "Error: code2flow is not installed. Please install code2flow from https://github.com/scottrogowski/code2flow."
     exit 1
 fi
 if ! command -v dot &> /dev/null; then
     echo "Error: Graphviz's dot command is not installed. Please install Graphviz from https://graphviz.org/download/."
     exit 1
 fi
 ```